### PR TITLE
Update Team per Recent Changes

### PR DESCRIPTION
**Pull request by ricardobaeta**
Signed-off-by: Ricardo Baeta <ricardo@ricardobaeta.com>

### CODE REVIEW

File: hugo/data/teams.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ sections:
         members:
         - picture: santiago.png
           name: Santiago Mola
-          position: Head of architecture
+          position: VP of Engineering
```
Comment (marnovo) [inline hugo/data/teams.yml]: Santiago should be in the Management team now.

File: hugo/data/teams.yml [hunk 2]
```diff
@@ -115,13 +118,16 @@ sections:
...
           name: Konstantin Slavnov
-          position: Junior Software Engineer
+          position: Software Engineer
           github: https://github.com/zurk
         - picture: waren.png
           name: Waren Long
           position: Junior Software Engineer
           github: https://github.com/warenlg
-
+        - picture: placeholder/male.png
```
Comment (marnovo) [inline hugo/data/teams.yml]: Missing Irina

File: hugo/data/teams.yml (hunk 1 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks @marnovo! Done.

File: hugo/data/teams.yml (hunk 2 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks @marnovo! Done.

File: hugo/data/teams.yml [hunk 3]
```diff
@@ -94,6 +76,21 @@ sections:
           name: Denys Smirnov
           position: Senior Software Engineer
           github: https://github.com/dennwc
+
+      - name: Data Processing Team
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: Given the ordering rules `Data Processing` should be before `Data Retrieval`

File: hugo/data/teams.yml [hunk 4]
```diff
@@ -94,6 +76,21 @@ sections:
...
+          github: https://github.com/ajnavarro          
+        - picture: kuba.png
+          name: Kuba Podgórski
+          position: Senior Software Engineer
+          github: https://github.com/kuba--
+        - picture: miguel.png
+          name: Miguel Molina
+          position: Senior Software Engineer
+          github: https://github.com/erizocosmico
         - picture: juanjo.png
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: Given the ordering rules Juanjo should be between Antonio and Kuba

File: hugo/layouts/engine/single.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
             <br><br>
             <h1 class="title text-left p-0 mt-5 mb-4">Turn your code into actionable insights with source{d} Engine</h1>
             <a href="https://go.sourced.tech/engine" class="btn btn-purple-light mr-4 mb-3 mb-sm-5">Request A Demo</a>
-            <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light mb-3 mb-sm-5">Install source{d} Engine</a>
+            <a href="https://docs.sourced.tech/engine#quickstart" class="btn btn-highlight-light mb-3 mb-sm-5">Install Engine</a>
```
Comment (dpordomingo) [inline hugo/layouts/engine/single.html]: Was this change done on purpose?

File: hugo/data/teams.yml [hunk 6]
```diff
@@ -181,3 +185,8 @@ sections:
         name: Francesc Campoy
         position: VP of Developer Relations
         github: https://github.com/campoy
+        - picture: santiago.png
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: I think the identation level should be reduced

File: hugo/data/teams.yml [hunk 7]
```diff
@@ -115,13 +112,20 @@ sections:
           github: https://github.com/m09
         - picture: konstantin.png
           name: Konstantin Slavnov
-          position: Junior Software Engineer
+          position: Software Engineer
           github: https://github.com/zurk
         - picture: waren.png
           name: Waren Long
           position: Junior Software Engineer
```
Comment (dpordomingo) [inline hugo/data/teams.yml]: I read that Waren was also promoted as Software Engineer.

File: hugo/data/teams.yml [hunk 8]
```diff
@@ -35,7 +29,7 @@ sections:
           github: https://github.com/smacker
         - picture: david.png
           name: David Pordomingo
-          position: Software Engineer
+          position: Senior Software Engineer
```
[the remark this answers is not in the document]
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks! Done.

File: hugo/data/teams.yml (hunk 3 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks! Done.

File: hugo/data/teams.yml (hunk 4 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks! Done.

File: hugo/layouts/engine/single.html (hunk 5 above)
Comment (ricardobaeta) [reply hugo/layouts/engine/single.html]: Yes, sorry about this. Reverted.

File: hugo/data/teams.yml (hunk 6 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Sorry, done!

File: hugo/data/teams.yml (hunk 7 above)
Comment (ricardobaeta) [reply hugo/data/teams.yml]: Thanks! Done.

**Comment by ricardobaeta** [comment]
Thank you so much for the detailed review @dpordomingo :)

@marnovo @smola How should I include Hassen? Thanks is advance!

**Comment by marnovo** [comment]
@ricardobaeta @dpordomingo  True, we need a new sales section w/ Hass + change the Dev Rel section to Mkt and add the new Alexander intern. Please is this is good as it is on the eng side (cc @smola), let's merge and deploy and open a new ticket for the remaining changes w/ me tagged and I'll provide the copy.

**Comment by dpordomingo** [comment]
ok, so let's wait for @smola approval of team section, as it can be seen at staging:
https://landing-staging.srcd.run/company/#team
I'll deploy once he approves the PR

**Comment by ricardobaeta** [comment]
@dpordomingo we're all good to go! Please be so kind as to merge and deploy to production!

**Comment by dpordomingo** [comment]
deployed into prod

**Comment by ricardobaeta** [comment]
Thanks @dpordomingo ✨

**Pull request merged**